flytekit/remote/remote.py

```diff
@@ -1144,8 +1144,10 @@ def _execute(
         """
         if execution_name is not None and execution_name_prefix is not None:
             raise ValueError("Only one of execution_name and execution_name_prefix can be set, but got both set")
-        execution_name_prefix = execution_name_prefix + "-" if execution_name_prefix is not None else None
-        execution_name = execution_name or (execution_name_prefix or "f") + uuid.uuid4().hex[:19]
+        if execution_name_prefix is not None:
+            execution_name = execution_name_prefix + "-" + uuid.uuid4().hex[:19]
+        if execution_name is None and execution_name_prefix is None:
+            execution_name = ""
```

Comment:
nit: we can remove these two lines, right? we should be able to pass none to the backend